documentation/asciidoc/computers/getting-started/configuring.adoc

```diff
@@ -1,49 +1,63 @@
-== Configuration on First Boot
+== Initial setup and configuration
 
-If you have not already configured your operating system using the xref:getting-started.adoc#advanced-options[Advanced Menu] of Raspberry Pi Imager when Raspberry Pi OS starts up for the first time, you will be guided through initial setup.
+When you start your Raspberry Pi for the first time after installing an operating system, a configuration wizard will guide you through the system's initial setup.
+
+NOTE: This wizard will not run if you used the xref:getting-started.adoc#advanced-options[Advanced options menu] of Raspberry Pi Imager to configure the operating system.
 
 .The Raspberry Pi Configuration Wizard dialog.
 image::images/initial-setup/1.png[alt=Click Next to get started with configuration.,width="80%"]
 
-The Raspberry Pi OS configuration wizard will run on the first boot. The wizard starts by allowing you to configure international settings and your timezone information.
+=== Set up Bluetooth devices
+
+If you're using a Bluetooth keyboard or mouse, it will tell you to put each device into pairing mode. While on this page, the Raspberry Pi will scan for pairable devices and connect to the first device it finds for each item. Your devices are ready to use when you see the `connected` dialog.
+
+This process works with the built-in Bluetooth adapter on Raspberry Pi 3 and 4, and with USB Bluetooth adapters on earlier models. Make sure any adapters you want to use are plugged in before powering on your Raspberry Pi.
```

Comment:
Since you can also use USB Bluetooth adaptors on those models of Pi that have built-in Bluetooth adaptors, this should probably be reworded. Perhaps something like: `Raspberry Pi 3 and 4 computers have built-in Bluetooth, but you can also use a USB Bluetooth adaptor.`